hyperapp.d.ts

```diff
@@ -0,0 +1,45 @@
+export as namespace hyperapp
+
+// VNode part
+
+export interface VNode<Data> {
```

Comment:
According to @pspeter3 DTS for Picodom, I am pretty sure this is to allow to define the Props interface of a vnode. And to help for JSX :tada:

![Props in VNode](http://i.imgur.com/YO0ZjF3.png)

_(VS Code is awesome regarding TypeScript)_